modules/create_timepoint/jsx/create_timepoint.js

```diff
@@ -0,0 +1,352 @@
+            {values[key]}
+          </label>
+        </div>
+      );
+    });
+    this.setState({errors: errors});
+  }
+
+  /**
+   * Handle form submission
+   * @param {object} e - Form submission event
+   */
+  handleSubmit(e) {
+    const state = Object.assign({}, this.state);
+    const send = this.urlSearchParams({
+      command: 'create',
+      candID: state.url.params.candID,
+      identifier: state.url.params.identifier,
+      subproject: state.form.value.subproject,
+      psc: state.form.value.psc,
+      visit: state.form.value.visit,
+    });
+    const url = this.props.dataURL + '/create_timepoint/AjaxTimepoint';
+    fetch(
+      url, {
+        method: 'POST',
+        mode: 'same-origin',
+        credentials: 'include',
+        redirect: 'follow',
+        headers: {
+          'Accept': 'application/json, text/plain, */*',
+          'Content-Type': 'application/x-www-form-urlencoded',
+        },
+        body: send,
+      }
+    ).then((response) => response.json())
+      .then(
+        (data) => {
+          if (data.status === 'error') {
+            // Populate the form errors.
+            if (data.errors) {
+              this.populateErrors(data.errors);
+            }
+          } else {
+            this.setState({success: true});
+          }
+    });
+  }
+  /**
+   * @return {DOMRect}
+   */
+  render() {
+    // Waiting for async data to load.
+    if (!this.state.isLoaded) {
+      return <Loader/>;
+    }
+    // Include form errors.
+    const errors = this.state.errors;
+    // Include subproject select.
+    const subproject = this.state.form.display.subproject ? (
+      <SelectElement
+        id={'subproject'}
+        name={'subproject'}
+        label={'Subproject'}
+        value={this.state.form.value.subproject}
+        options={this.state.form.options.subproject}
+        onUserInput={this.setForm}
+        emptyOption={false}
+        disabled={false}
+        required={true}
+      />
+    ) : '';
+    // Include psc select.
+    const psc = this.state.form.display.psc ? (
+      <SelectElement
+        id={'psc'}
+        name={'psc'}
+        label={'Site'}
+        value={this.state.form.value.psc}
+        options={this.state.form.options.psc}
+        onUserInput={this.setForm}
+        emptyOption={false}
+        disabled={false}
+        required={true}
+      />
+    ) : '';
+    // Include visit select.
+    const visit = this.state.form.display.visit ? (
+      <SelectElement
+        id={'visit'}
+        name={'visit'}
+        label={'Visit label'}
+        value={this.state.form.value.visit}
+        options={this.state.form.options.visit}
+        onUserInput={this.setForm}
+        emptyOption={false}
+        disabled={false}
+        required={true}
+      />
+    ) : '';
+
+    if (!this.state.success) {
+      return (
+        <div>
+          <div>
+            <h3>Create Time Point</h3> <br/>
+            {errors}
+            <FormElement
```

Comment:
to keep consistent with recent UX group UI decisions, I would suggest nesting the <FormElement /> with a <Panel /> element with a title prop "Create Time Point" and remove the h3 tag